allennlp/training/trainer.py

```diff
@@ -30,10 +30,28 @@
 from allennlp.training.moving_average import MovingAverage
 from allennlp.training.optimizers import Optimizer
 from allennlp.training.tensorboard_writer import TensorBoardWriter
+from allennlp.training.log_writer import LogWriter
 
 logger = logging.getLogger(__name__)
 
 
+def get_train_and_validation_metrics(metrics: Dict):
```

Comment:
Nit: should probably be a private function:
```suggestion
def _get_train_and_validation_metrics(metrics: Dict):
```